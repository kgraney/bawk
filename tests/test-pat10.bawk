
LE = 0;
/ "IHDR" / {
	print(100);
	print(RP);
}

LE = 1;
/ "IHDR" / {
	print(200);
	print(RP);
}

