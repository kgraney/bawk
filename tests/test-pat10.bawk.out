100
16
200
16
